spec/features/global_announcements_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'Global announcements', type: :feature do
+  subject { page }
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    describe 'no global announcements' do
+      before do
+        instance.announcements.clear
```

Comment:
Don't think there's any confusion, it's pretty convention, when tests added which does not require that precondition you should do :
```ruby
describe 'precondition' do
  before { some precondition }
  context 'xxx' do
  end

  context 'xx' do
  end
end

describe 'other preconditions or no precondition' do
  context 'xxx' 
end
```